meta-schema.graphql

```diff
@@ -0,0 +1,115 @@
+# Base types
+type Party {
+	id:ID!
+	idType:String! #PartyId, IPI, ISNI, MBID
+	displayName:String!
+	firstName: String
+	lastName: String
+}
+
+type Organisation {
+	id:String!
+	idType:String! #DPid, IPI, MBID
+	displayName:String!
+	formalName:String
+}
+
+# Music Types
+type TerritoryAgreement {
+	territoryCode:Int! #CISAC Territory standard
+	condition:String! # CWR inclusion/exclusion indicator
+}
+
+type InterestedPartyAgreement {
+	agreementRoleCode:String!
+	interestedPartyNumber:Int!
+	ipi:Int
+	ipiBase:Int
+	lastName:String!
+	firstName:String
+	prSociety:String
+	prShare:Int
+	mrSociety:String
+	mrShare:Int
+}
+
+type MusicWork {
+	iswc:String
+	workTitle:String!
+	territories:[TerritoryAgreement]
+	interestedParty:[InterestedPartyAgreement]
+	publishers:[Organisation]
+	composers:[MusicComposer]
+	recordings:[MusicRecording]
+}
+
+# DDEX:ERN:SoundRecording
+type MusicRecording {
+	isrc: String
+	iswc: String
+	displayTitle:String!
+	displayArtist:String!
+	contributors:[Party]
+	label:[Organisation]
+	genre:String
+	work:MusicWork
+	products:[MusicProduct]
+}
+
+type TerritoryProduct {
+	territoryCode:String!
```

Comment:
Is this the same code as it used in `TerritoryAgreement`? If so, they should probably have the same type.

Reply:
Good catch. In CWR they use a different territory type to ERN. ERN uses ISO territory codes, CWR using CISAC territory standard. So it is a question of preferred option. Personally i think ISO codes are wider known, but happy to discuss.

Reply:
Ok, so maybe this should be named `isoTerritoryCode` to make it distinct from `cisacTerritoryCode`?